integration-fe/src/sagas/getAppName.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-debugger */
+import { call, put } from 'redux-saga/effects';
+import { getAppName as appName } from './apiCalls';
```

Comment:
This is part of react-saga. The "call" effect calls an api end point, or an api service. If the call executes an async call, it pauses the execution until the async operation has finished.
After the "call" to be completed, we dispatch an action with the "put" effect.